### PR TITLE
[Main] [Release] Increase version to 3.1.0

### PR DESCRIPTION
**Pull request by MSNev**

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -2,25 +2,78 @@
 
 > Note: ES3/IE8 compatibility will be removed in the future v3.x.x releases (scheduled for mid-late 2022), so if you need to retain ES3 compatibility you will need to remain on the 2.x.x versions of the SDK or your runtime will need install polyfill's to your ES3 environment before loading / initializing the SDK.
 
-## 3.1.0 (Feb ??, 2024)
+## 3.1.0 (Feb 14t, 2024)
```
Comment (Karlie-777) [inline RELEASES.md:5]: nit: Feb 14th

File: examples/shared-worker/package.json [hunk 2]
```diff
@@ -63,12 +63,9 @@
     "dependencies": {
         "@microsoft/applicationinsights-shims": "3.0.1",
         "@microsoft/dynamicproto-js": "^2.0.3",
-        "@microsoft/applicationinsights-web": "3.0.8",
-        "@microsoft/applicationinsights-web-snippet": "1.1.6",
-        "@microsoft/applicationinsights-core-js": "3.0.8",
+        "@microsoft/applicationinsights-web": "3.1.0",
+        "@microsoft/applicationinsights-web-snippet": "1.1.0",
```
Comment (MSNev) [inline examples/shared-worker/package.json:67]: The last released version of this package was 1.0.1, so rolling back this number to the next release

File: tools/applicationinsights-web-snippet/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/applicationinsights-web-snippet",
-    "version": "1.1.6",
+    "version": "1.1.0",
```
Comment (MSNev) [inline tools/applicationinsights-web-snippet/package.json:3]: The last released version of this package was 1.0.1, so rolling back this number to the next release

File: version.json [hunk 4]
```diff
@@ -1,104 +1,104 @@
...
         },
         "@microsoft/1ds-core-js": {
             "package": "shared/1ds-core-js/package.json",
-            "release": "4.0.6"
+            "release": "4.1.0"
         },
         "@microsoft/applicationinsights-web-snippet": {
             "package": "tools/applicationinsights-web-snippet/package.json",
-            "release": "1.1.6"
+            "release": "1.1.0"
```
Comment (MSNev) [inline version.json:101]: The last released version of this package was 1.0.1, so rolling back this number to the next release

**Review by Karlie-777 — APPROVED** [review]

**Pull request merged**